src/lib/reasoners/intervals.mli

```diff
@@ -1,7 +1,7 @@
 (**************************************************************************)
 (*                                                                        *)
 (*     Alt-Ergo: The SMT Solver For Software Verification                 *)
-(*     Copyright (C) 2013-2023 --- OCamlPro SAS                           *)
+(*     Copyright (C) 2024 --- OCamlPro SAS                                *)
```

Comment:
Please don't remove `2013` here. Alt-Ergo wasn't the property of OCamlPro before this date.